packages/atomic/src/components/atomic-query-error/atomic-query-error.tsx

```diff
@@ -16,27 +29,94 @@ export class AtomicQueryError implements InitializableComponent {
   @InitializeBindings() public bindings!: Bindings;
   public queryError!: QueryError;
 
+  @BindStateToI18n()
+  @State()
+  private strings = {
+    noEndpoints: () => this.bindings.i18n.t('noEndpoints'),
+    invalidToken: () => this.bindings.i18n.t('invalidToken'),
+    addSources: () => this.bindings.i18n.t('addSources'),
+    coveoOnlineHelp: () => this.bindings.i18n.t('coveoOnlineHelp'),
+    cannotAccess: () => this.bindings.i18n.t('cannotAccess'),
+    somethingWentWrong: () => this.bindings.i18n.t('somethingWentWrong'),
+    ifProblemPersists: () => this.bindings.i18n.t('ifProblemPersists'),
```

Comment:
I think it would be more flexible to name the key according to the end-use case. If the message changes, the key will still be logical.

e.g.
`somethingWentWrong` -> `queryErrorMessageTitle`
`ifProblemPersists` -> `queryErrorMessageDescription`